server/auth.go

```diff
@@ -57,6 +57,7 @@ type Account struct {
 	maxnae   int
 	maxaettl time.Duration
 	pruning  bool
+	checksti bool // check stream imports during a config reload
```

Comment:
I evaluate at top of reloadAuthorization() under server lock. I then need to know on a per client basis when processing client's subscriptions. I could (in reloadAuthorization()) create a map[string]bool and pass that to processSubsOnConfigReload() where I could do a lookup instead of checking account's field. Advantage too, is that I would not have to clear the boolean at end of reloadAuthorization().